tools/osbuilder/node-builder/azure-linux/uvm_build.sh

```diff
@@ -38,39 +87,7 @@ fi
 
 pushd "${repo_dir}"
 
-echo "Building rootfs and including pre-built agent binary"
-pushd tools/osbuilder
-# This command requires sudo because of dnf-installing packages into rootfs. As a suite, following commands require sudo as well as make clean
-sudo -E PATH=$PATH make ${rootfs_make_flags} -B DISTRO=cbl-mariner rootfs
-ROOTFS_PATH="$(readlink -f ./cbl-mariner_rootfs)"
-popd
-
-echo "Installing agent service files into rootfs"
-sudo cp ${AGENT_INSTALL_DIR}/usr/lib/systemd/system/kata-containers.target ${ROOTFS_PATH}/usr/lib/systemd/system/kata-containers.target
-sudo cp ${AGENT_INSTALL_DIR}/usr/lib/systemd/system/kata-agent.service ${ROOTFS_PATH}/usr/lib/systemd/system/kata-agent.service
-
-if [ "${CONF_PODS}" == "yes" ]; then
-	echo "Building tarfs kernel driver and installing into rootfs"
-	pushd src/tarfs
-	make KDIR=${UVM_KERNEL_HEADER_DIR}
-	sudo make KDIR=${UVM_KERNEL_HEADER_DIR} KVER=${UVM_KERNEL_VERSION} INSTALL_MOD_PATH=${ROOTFS_PATH} install
-	popd
-
-	echo "Building dm-verity protected image based on rootfs"
-	pushd tools/osbuilder
-	sudo -E PATH=$PATH make DISTRO=cbl-mariner MEASURED_ROOTFS=yes DM_VERITY_FORMAT=kernelinit image
-	popd
-
-	echo "Building IGVM and UVM measurement files"
-	pushd tools/osbuilder
-	sudo chmod o+r root_hash.txt
-	sudo make igvm DISTRO=cbl-mariner IGVM_SVN=${IGVM_SVN}
-	popd
-else
-	echo "Building image based on rootfs"
-	pushd tools/osbuilder
-	sudo -E PATH=$PATH make DISTRO=cbl-mariner image
-	popd
-fi
+build_uvm "release"
```

Comment:
can also think if we change this caller of this instead of executing both paths here in one script